src/backend/storeManagers/hyperplay/library.ts

```diff
@@ -206,10 +207,12 @@ export async function listUpdateableGames(): Promise<string[]> {
         To continue to receive game updates, uninstall and reinstall this game: ${val.title}`)
       }
       if (
+        // games installed before 0.5.0 used gameInfo.version for install.version
+        // so for backwards compatibility we remove spaces and lower case channel release meta name here
         val.install.version !==
-          val.channels[val.install.channelName].release_meta.name
-            ?.toLowerCase()
-            .replaceAll(' ', '') ||
+          sanitizeVersion(
+            val.channels[val.install.channelName].release_meta.name
+          ) ||
```

Comment:
I'm curious what the `|| ''` is used for 🤔

Reply:
you're right. it should be moved next to `val.channels[val.install.channelName].release_meta.name` because that can come back undefined and then sanitizeVersion will throw.

Fixed now